mcs/class/corlib/corlib.dll.sources

```diff
@@ -1676,6 +1677,9 @@ corert/RuntimeAugments.cs
 
 ../../../external/corefx/src/Common/src/CoreLib/System/Reflection/BindingFlags.cs
 
+../../../external/corefx/src/Common/src/CoreLib/Interop/Unix/System.Native/Interop.GetRandomBytes.cs
```

Comment:
This will break on Windows as we don't build System.Native there. You'll need to include this only for Unix profiles and include `external/corefx/src/Common/src/CoreLib/Interop/Windows/BCrypt/Interop.BCryptGenRandom.cs` on Windows instead (like System.dll does it with similar Interop files).